cg/cli/add.py

```diff
@@ -226,10 +226,10 @@ def case(
         LOG.error(f"{customer_id}: customer not found")
         raise click.Abort
 
-    for panel_id in panels:
-        panel: Panel = status_db.panel(abbrev=panel_id)
+    for panel_abbreviation in panels:
```

Comment:
I agree but I don't want to rename it since the name is tied to the CLI command. We should create a separate issue for going through and improving the naming of the CLI flags to accurately describe what they actually are.